test/resources/examples/portal/src/mocks/enableApiMocking.ts

```diff
@@ -21,9 +21,7 @@ export async function enableApiMocking() {
     if (localStorage.getItem('enableMockApi') === 'true') {
         // Start the mock service worker
         try {
-            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-            // @ts-ignore - This is a dynamic import to create the mocks
-            const { worker } = (await import('./browser.js')) as { worker: SetupWorker };
```

Comment:
Does this work? We made this as a workaround and it goes together with this alias
```
'./browser.js': Path.resolve(__dirname, './src/mocks/.generated/browser.ts'),
```
in `webpack.development.config.js.hbs`.

Reply:
<img width="1442" alt="image" src="https://github.com/kapetacom/codegen-target-react-ts/assets/1045799/c6df3221-6760-4198-84c0-b2c70ca432e8">

Reply:
Ah ok - didn't notice the alias. My build failed b ut could be because I dont have the alias

Reply:
I'm not getting that error - where are you seeing that ?